Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -1236,16 +1244,20 @@ public bool TryAppendLineSafely(CurveArray curveArray, Line line, ApplicationObj
       appObj.Log.Add("Some lines in the CurveArray where ignored due to being smaller than the allowed curve length.");
       return false;
     }
+#pragma warning disable CA1031 // Do not catch general exception types
     try
     {
       curveArray.Append(LineToNative(line));
       return true;
     }
-    catch (Exception e)
+    catch (Exception ex)
     {
```

Comment:
Another one I'm not sure we should silence.

If the issue is that we don't have a known exception that can be thrown from `LineToNative`, then we should pull on that thread and find out.